### ISSUE TITLE
no-duplicate-case is buggy with object properties

### ISSUE THREAD

**Issue by fflorent** [eslint/eslint#2410]
Test case : 
1. Create the file foo.js (below)
2. Run eslint on this file (the `no-duplicate-case` should be enabled)

Expected result: There should be no error

Actual result: error with the `no-duplicate-case` rule

Using Eslint 0.17.0

foo.js : 
````javascript
/* eslint no-console: 0 */
/* global console: 1 */
var STATUSES = {
	FOO: 42,
	BAR: 43
};
var b = 42;

switch (b) {
	case STATUSES.FOO:
		console.log("foo");
		break;
	case STATUSES.BAR:
		console.log("bar");
		break;
}
````

Florent

**Comment by fflorent** [comment]
Sorry, duplicate of this one: #2038.

Florent
